x-pack/legacy/plugins/ml/server/routes/results_service.ts

```diff
@@ -0,0 +1,220 @@
+ */
+export function resultsServiceRoutes({ xpackMainPlugin, router }: RouteInitialization) {
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {post} /api/ml/results/anomalies_table_data Prepare anomalies records for table display
+   * @apiName GetAnomaliesTableData
+   * @apiDescription Retrieves anomaly records for an anomaly detection job and formats them for anomalies table display
+   */
+  router.post(
+    {
+      path: '/api/ml/results/anomalies_table_data',
+      validate: {
+        body: schema.object({ ...anomaliesTableDataSchema }),
+      },
+    },
+    licensePreRoutingFactory(xpackMainPlugin, async (context, request, response) => {
+      try {
+        const resp = await getAnomaliesTableData(context, request.body);
+
+        return response.ok({
+          body: resp,
+        });
+      } catch (e) {
+        return response.customError(wrapError(e));
+      }
+    })
+  );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {post} /api/ml/results/category_definition Returns category definition
+   * @apiName GetCategoryDefinition
+   * @apiDescription Returns the definition of the category with the specified ID and job ID
+   */
+  router.post(
+    {
+      path: '/api/ml/results/category_definition',
+      validate: {
+        body: schema.object({
+          jobId: schema.maybe(schema.string()),
+          categoryId: schema.string(),
+        }),
+      },
+    },
+    licensePreRoutingFactory(xpackMainPlugin, async (context, request, response) => {
+      try {
+        const resp = await getCategoryDefinition(context, request.body);
+
+        return response.ok({
+          body: resp,
+        });
+      } catch (e) {
+        return response.customError(wrapError(e));
+      }
+    })
+  );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {post} /api/ml/results/max_anomaly_score Returns the maximum anomaly_score
+   * @apiName GetMaxAnomalyScore
+   * @apiDescription Returns the maximum anomaly_score for result_type:bucket over jobIds for the interval passed in
+   */
+  router.post(
+    {
+      path: '/api/ml/results/max_anomaly_score',
+      validate: {
+        body: schema.object({
+          jobIds: schema.arrayOf(schema.string()),
+          earliestMs: schema.number(),
+          latestMs: schema.number(),
+        }),
+      },
+    },
+    licensePreRoutingFactory(xpackMainPlugin, async (context, request, response) => {
+      try {
+        const resp = await getMaxAnomalyScore(context, request.body);
+
+        return response.ok({
+          body: resp,
+        });
+      } catch (e) {
+        return response.customError(wrapError(e));
+      }
+    })
+  );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {post} /api/ml/results/category_examples Returns the maximum anomaly_score
+   * @apiName GetCategoryExamples
+   * @apiDescription Returns the categorization examples for the categories with the specified IDs from the given index and job ID
```

Comment:
Edit to `Returns examples for the categories with the specified IDs from the job with the supplied ID.`

Reply:
Updated 0574d51d9d2b613b8d0c7ecf60385eddb433de33